ajax/libs/vue2-leaflet/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "vue2-leaflet",
```

Comment:
We took GitHub as the source of auto-update config here. Let's use `Vue2Leaflet` instead.
Don't forget to revise folder name too.
Thanks.